gst/nnstreamer/tensor_query/tensor_query_serversink.c

```diff
@@ -134,6 +134,79 @@ gst_tensor_query_serversink_finalize (GObject * object)
   G_OBJECT_CLASS (parent_class)->finalize (object);
 }
 
+/**
+ * @brief start processing of query_serversink
+ */
+static void
+_gst_tensor_query_serversink_start (GstTensorQueryServerSink * sink)
+{
+  g_autofree gchar *id_str = NULL;
```

Comment:
I am not sure about using ```g_autofree``` for the function arguments. 

https://stackoverflow.com/questions/11446136/c-auto-ptr-in-functions-as-argument-and-return-value

Reply:
Thanks, Changed!